manifests/server/puppetdb.pp

```diff
@@ -23,17 +23,23 @@
 #
 # @param soft_write_failure
 #   Whether to enable soft write failure
+#
+# @param terminus_package
+#   The PuppetDB terminus package
+#
 class puppet::server::puppetdb (
   Stdlib::Host $server = undef,
   Stdlib::Port $port = 8081,
   Boolean $soft_write_failure = false,
+  String[1] $terminus_package = 'puppetdb-termini',
```

Comment:
https://github.com/theforeman/puppet-puppet/pull/936 is open for this same change and there I also suggested to make it `Optional`
```suggestion
  Optional[String[1]] $terminus_package = undef,
```